frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx

```diff
@@ -429,6 +443,78 @@ function NewChannelInternal({
           </ExternalLinkButton>
         </div>
       </div>
+
+      {/* Confirmation Modal */}
+      <Dialog open={showConfirmModal} onOpenChange={setShowConfirmModal}>
+        <DialogContent className="sm:max-w-md">
+          <DialogHeader>
+            <DialogTitle>Confirm Channel Opening</DialogTitle>
+            <DialogDescription>
+              Are you sure you want to open a Lightning channel with the
+              following details?
+            </DialogDescription>
+          </DialogHeader>
+
+          <div className="space-y-4">
+            <div className="grid grid-cols-2 gap-4 text-sm">
+              <div>
+                <div className="font-medium text-muted-foreground">Peer</div>
+                <div>{selectedPeer?.name || "Custom"}</div>
+              </div>
+              <div>
+                <div className="font-medium text-muted-foreground">Amount</div>
+                <div>{formatAmount(parseInt(order.amount || "0"))} sats</div>
+              </div>
+              <div>
+                <div className="font-medium text-muted-foreground">
+                  Channel Type
+                </div>
+                <div>{order.isPublic ? "Public" : "Private"}</div>
+              </div>
+              <div>
+                <div className="font-medium text-muted-foreground">
+                  Payment Method
+                </div>
+                <div>On-chain</div>
+              </div>
+            </div>
+
+            {selectedPeer?.name === "Custom" && order.pubkey && (
+              <div className="text-sm">
+                <div className="font-medium text-muted-foreground">
+                  Node Public Key
+                </div>
+                <div className="font-mono text-xs break-all bg-muted p-2 rounded">
+                  {order.pubkey}
+                </div>
+              </div>
+            )}
+
+            <div className="p-4 bg-orange-50 dark:bg-orange-950/20 rounded-lg border border-orange-200 dark:border-orange-800">
+              <div className="text-sm text-orange-800 dark:text-orange-200">
+                <strong>Important:</strong> Opening a channel requires an
+                on-chain transaction and network fees. This action cannot be
+                undone. Please verify all details before proceeding.
+              </div>
+            </div>
+          </div>
+
+          <DialogFooter className="gap-2">
+            <Button
+              variant="outline"
+              onClick={() => setShowConfirmModal(false)}
+            >
+              Cancel
+            </Button>
+            <Button
+              onClick={handleConfirmSubmit}
+              className="bg-orange-500 hover:bg-orange-600"
```

Comment:
We should use a variant here rather than custom colors. But maybe the default variant is fine.